src/content/learn/start-a-new-react-project.md

```diff
@@ -1,130 +1,129 @@
 ---
-title: Start a New React Project
+title: Старт нового проєкту на React
 ---
 
 <Intro>
 
-If you want to build a new app or a new website fully with React, we recommend picking one of the React-powered frameworks popular in the community.
+Якщо ви вирішили розробити новий застосунок або новий вебсайт цілком за допомогою React, ми радимо обрати один зі фреймворків, заряджених React, популярних у нашій спільноті.
 
 </Intro>
 
+React можна користуватися без фреймворку, проте ми з'ясували, що більшість застосунків і сайтів, зрештою, розробляють рішення для поширених проблем, наприклад, розбиття коду, маршрутизації, отримання даних і генерування HTML. Ці проблеми – спільні для всіх бібліотек UI, а не лише для React.
```

Comment:
Please retain two empty lines before this paragraph, as we need line-by-line parity between the original and translated text.